packages/cozy-dataproxy-lib/README.md

```diff
@@ -28,3 +30,89 @@ In order to create a SearchEngine:
   const query = 'Some search string'
   const searchResult = searchEngine.search(query)
 ```
+
+## Search and AI components
+
+### Prerequisites
+
+0. Update `cozy-scripts` version of your app to 8.4.0.
+
+1. Add following permissions in manifest.webapp :
+
+```json
+"chatConversations": {
+  "description": "Required by the cozy Assistant",
+  "type": "io.cozy.ai.chat.conversations",
+  "verbs": ["GET", "POST"]
+},
+"chatEvents": {
+  "description": "Required by the cozy Assistant",
+  "type": "io.cozy.ai.chat.events",
+  "verbs": ["GET"]
+}
+```
+
+2. Add the provider in your tree :
+
+```jsx
+<DataProxyProvider>
+  children
+</DataProxyProvider>
+```
+
+3. Add realtime queries for chat conversations in your tree :
+
+```jsx
+<RealTimeQueries doctype="io.cozy.ai.chat.conversations" />
+```
```

Comment:
The 1. and 3. steps are only for AI assistant, not required for search, we should separate the requirements